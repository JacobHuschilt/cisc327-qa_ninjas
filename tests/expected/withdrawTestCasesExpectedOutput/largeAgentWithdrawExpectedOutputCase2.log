Thank you for your withdraw!