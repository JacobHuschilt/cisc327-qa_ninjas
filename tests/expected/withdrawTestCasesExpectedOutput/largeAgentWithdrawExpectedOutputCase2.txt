WDR 0000000 999999999 1234567
EOS 0000000 000 0000000 ***